apps/infrastructure/src/iam_policy.tf

```
@@ -160,7 +160,8 @@ resource "aws_iam_policy" "deploy_website" {
         Effect = "Allow"
         Resource = concat(
           [aws_cloudfront_distribution.website.arn],
-          [for name, distribution in aws_cloudfront_distribution.cdn_multi_website : distribution.arn]
+          [for name, distribution in aws_cloudfront_distribution.cdn_multi_website : distribution.arn],
+          [aws_cloudfront_distribution.storybook.arn]
```

If you agree, I would put the storybook distribution in the first list like the following:

```suggestion
          [aws_cloudfront_distribution.website.arn, aws_cloudfront_distribution.storybook.arn],
          [for name, distribution in aws_cloudfront_distribution.cdn_multi_website : distribution.arn]
```